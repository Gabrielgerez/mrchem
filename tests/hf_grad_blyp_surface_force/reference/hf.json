{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "r_rms": 5.2927138943e-05,
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_per_node": -1,
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000,
      "use_omp_num_threads": false
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "blyp"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (BLYP)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "rotate": true,
        "screen": 12.0,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "geometric_derivative": {
          "geom-1": {
            "method": "surface_integrals",
            "operator": "h_nuc_grad",
            "precision": 0.001,
            "radius_factor": 0.6,
            "smoothing": 0.001,
            "surface_integral_precision": "medium"
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655945
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.7062277013424665,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.8876650936712203,
            -0.28715378402776714,
            1.4285679384000591
          ],
          "vector_el": [
            22.594679000503028,
            4.619919686231877,
            -6.462430480102103
          ],
          "vector_nuc": [
            -23.48234409417425,
            -4.907073470259644,
            7.890998418502162
          ]
        }
      },
      "geometric_derivative": {
        "geom-1": {
          "electronic": [
            1.8050305045278834,
            0.32708582380145046,
            -2.827035914174666,
            -1.1865116586891244,
            -0.3689310993139891,
            1.89962774450718
          ],
          "electronic_norm": 4.0632201464342454,
          "nuclear": [
            -1.5689195606619417,
            -0.49421531749272585,
            2.5173387449218514,
            1.5689195606619417,
            0.49421531749272585,
            -2.5173387449218514
          ],
          "nuclear_norm": 4.252705427032731,
          "total": [
            0.23611094386594167,
            -0.1671294936912754,
            -0.30969716925281476,
            0.38240790197281727,
            0.12528421817873675,
            -0.6177110004146713
          ],
          "total_norm": 0.8503480469591129
        }
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -16.00402622580678,
          4.277122522973494,
          4.733981539009209,
          4.7644909520757075,
          5.0688752263655426
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": 5.680888029234348
      },
      "scf_energy": {
        "E_ee": 94.63986905851992,
        "E_eext": 0.0,
        "E_el": -84.28375952895213,
        "E_en": -329.6603919625763,
        "E_kin": 166.8378397604974,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -79.0814473879332,
        "E_x": 0.0,
        "E_xc": -16.101076385393156,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.63986905851992,
        "E_eext": 0.0,
        "E_el": -84.28375952895213,
        "E_en": -329.6603919625763,
        "E_kin": 166.8378397604974,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -79.0814473879332,
        "E_x": 0.0,
        "E_xc": -16.101076385393156,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
