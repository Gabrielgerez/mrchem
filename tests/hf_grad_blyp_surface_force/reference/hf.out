                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         master                                   ***
***         Git commit hash    60b28085698bcbe7db1f-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Fri Oct 10 11:40:35 2025 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : master
 Git commit hash       : f0000a79200044229a5d
 Git commit author     : gitpeterwind
 Git commit date       : Fri Oct 10 09:04:34 2025 +0200

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.996812789590e-01
    1    2    p          :                              9.998694293756e-01
    2    2    p          :                              9.999998231017e-01
    3    2    p          :                              9.999999836389e-01
    4    2    p          :                              9.999998001090e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (BLYP)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.869957524159
 E-N energy              :            (au)               -329.770913433706
 Coulomb energy          :            (au)                 94.651043608099
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -16.081758084932
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -87.331670386380
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.212935824536e+01
                         :      (kcal/mol)             -5.153695039768e+04
                         :        (kJ/mol)             -2.156306004639e+05
                         :            (eV)             -2.234853689350e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -16.188997304518
    1    2    p          :            (au)                  3.197062117683
    2    2    p          :            (au)                  4.744873362651
    3    2    p          :            (au)                  4.767570460468
    4    2    p          :            (au)                  4.768191191106
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  2.577399654781
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.869957524159
 E-N energy              :            (au)               -329.770913433706
 Coulomb energy          :            (au)                 94.651043608099
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -16.081758084932
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -87.331670386380
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.212935824536e+01
                         :      (kcal/mol)             -5.153695039768e+04
                         :        (kJ/mol)             -2.156306004639e+05
                         :            (eV)             -2.234853689350e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -16.188997304518
    1    2    p          :            (au)                  3.197062117683
    2    2    p          :            (au)                  4.744873362651
    3    2    p          :            (au)                  4.767570460468
    4    2    p          :            (au)                  4.768191191106
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  2.577399654781
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       22.585137        4.622958       -6.451441
 Magnitude               :            (au)                       23.939115
                         :         (Debye)                       60.847162
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189426
---------------------------------------------------------------------------
 Total vector            :       -0.897207       -0.284116        1.439558
 Magnitude               :            (au)                        1.719892
                         :         (Debye)                        4.371529
===========================================================================


===========================================================================
                       Geometric Derivative (geom-1)
---------------------------------------------------------------------------
 Total                   :        0.126052        0.067085       -0.242930
                         :        0.408234        0.128509       -0.652749
 Norm                    :            (au)                        0.829853
---------------------------------------------------------------------------
 Nuclear                 :       -1.568920       -0.494215        2.517339
                         :        1.568920        0.494215       -2.517339
 Norm                    :            (au)                        4.252705
---------------------------------------------------------------------------
 Electronic              :        1.694971        0.561300       -2.760268
                         :       -1.160685       -0.365706        1.864589
 Norm                    :            (au)                        3.970478
---------------------------------------------------------------------------
 Est. var. of force error:            (au)                      4.3320e-01
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
